skops/cli/_convert.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle
+from typing import Optional
+
+from skops.cli._utils import get_log_level
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert_file(
+    input_file: os.PathLike,
+    output_file: os.PathLike,
+    logger: logging.Logger = logging.getLogger(),
```

Comment:
Hmm, we could, although that would mean it gets initialised even if we import functions from the file to use elsewhere.

I don't think it would really impact much, I just personally dislike writing anything other than definitions at the top level

Reply:
> that would mean it gets initialised even if we import functions from the file to use elsewhere.

That's true. On the other hand it's now being initialized each time the function is called. Given that CLI usage consists of creating a new process anyway, it makes indeed no difference either way. If we ever get to a point where we use the logger elsewhere, we might have to rethink this.